studio/src/mas-repository.js

```diff
@@ -314,36 +339,57 @@ export class MasRepository extends LitElement {
         Store.fragments.recentlyUpdated.loading.set(false);
     }
 
+    /**
+     * Helper method to create fragment fields from data object
+     * @param {Object} data - The data object containing field values
+     * @param {Array} existingFields - Any existing fields to include
+     * @returns {Array} The complete fields array
+     */
+    createFieldsFromData(data, existingFields = []) {
+        if (!data) return existingFields;
+        
+        return Object.entries(data)
+            .filter(([key, value]) => value !== undefined)
+            .reduce((fields, [key, value]) => {
+                const type = key === 'locReady' ? 'boolean' : 'text';
+                fields.push({ name: key, type, values: [value] });
+                return fields;
+            }, [...existingFields]);
+    }
+
     async createFragment(fragmentData) {
         try {
-            Events.toast.emit({
-                variant: 'info',
-                content: 'Creating fragment...',
-            });
+            const isPlaceholder = fragmentData.data && (fragmentData.data.key !== undefined || 
+                                                       fragmentData.parentPath?.includes('/dictionary/'));
+            this.showToast(isPlaceholder ? 'Creating placeholder...' : 'Creating fragment...');
```

Comment:
i think you can just leave it as create Fragment, that's fine

Reply:
done